tests/e2e/test-e2e-performance.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo "🎭 Running Playwright e2e tests in default headless mode, skipping @todo.";
```

Comment:
Would it be worthwhile to indicate that we are running the e2e _performance_ tests here?


```suggestion
echo "🎭 Running Playwright e2e performance tests in default headless mode, skipping @todo.";
```

Reply:
I think it's nice to have that. Updated the copy on [1878b24](https://github.com/Automattic/woocommerce-payments/pull/10352/commits/1878b24011a715f6a714f226af5a17eae50f6713).